tasks/gulp/__tests__/after-build-package.test.js

```diff
@@ -89,4 +98,42 @@ describe('package/', () => {
       await sassRender({ file: allScssFile })
     })
   })
+
+  describe('components', () => {
+    it('should each have macro-options.json that contains JSON', done => {
+      const componentNames = lib.allComponents.slice()
+
+      const getMacroOptionsContents = () => {
+        return componentNames.map(name => {
+          const filePath = path.join(configPaths.package, 'components', name, 'macro-options.json')
+          return readFile(filePath, 'utf8')
+            .then((data) => {
+              var newData = JSON.parse(data)
+              // We expect the component JSON to contain at least "name" and "type"
+              expect(newData).toEqual(
+                expect.arrayContaining([
+                  expect.objectContaining({
+                    name: expect.anything(),
+                    type: expect.anything(),
+                    required: expect.anything(),
+                    description: expect.anything()
+                  })
+                ])
+              )
+            }).catch(error => {
+              throw error
+            })
+            .then(data => {
+              return { data }
+            }).catch(error => {
+              throw error
+            })
+        })
+      }
+      Promise
+        .all(getMacroOptionsContents())
+        .then(() => { done() })
+        .catch(error => { throw error })
```

Comment:
don't think this line is needed as the test fails higher up in the getMacroOptions function

Reply:
Removed!